typings/index.d.ts

```diff
@@ -3062,7 +3062,7 @@ interface FetchInviteOptions extends BaseFetchOptions {
 }
 
 interface FetchInvitesOptions {
-  channelID?: Snowflake;
+  channelId?: GuildChannelResolvable;
```

Comment:
Mmh, no? `channelId` is a Snowflake, not a resolvable. It's a good idea to change this to a `GuildChannelResolvable` but this means that the name should be changed to `channel` and `resolveId` should be changed to `resolve` in the function.